test/unit/three.source.unit.js

```diff
@@ -86,6 +86,7 @@ import './src/extras/curves/LineCurve3.tests.js';
 import './src/extras/curves/QuadraticBezierCurve.tests.js';
 import './src/extras/curves/QuadraticBezierCurve3.tests.js';
 import './src/extras/curves/SplineCurve.tests.js';
+import './src/extras/curves/NURBSCurve.tests.js';
```

Comment:
Add an entry to `test/unit/three.addons.unit.js` instead.

Reply:
Done